jbpm-console-ng-case-mgmt/jbpm-console-ng-case-mgmt-backend/src/main/java/org/jbpm/console/ng/cm/backend/server/CaseActionAdhocFragmentMapper.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jbpm.console.ng.cm.backend.server;
+
+
+import java.util.function.Function;
+
+import org.jbpm.console.ng.cm.model.CaseActionSummary;
+import org.kie.server.api.model.cases.CaseAdHocFragment;
+
+public class CaseActionAdhocFragmentMapper implements Function<CaseAdHocFragment, CaseActionSummary> {
+
+    @Override
+    public CaseActionSummary apply(CaseAdHocFragment adHocFragment) {
+        if (adHocFragment == null) {
+            return null;
+        }
+
+        return CaseActionSummary.builder()
+                .name(adHocFragment.getName())
+                .status("AdhocFragment")
```

Comment:
Is this hardcoded string going to be displayed anywhere? I don't see any value in setting it to hardcoded string. If status doesn't make sense for ad hoc fragments I'd rather see something line "Does not apply" or "N/A"